src/novi/basics/Main.java

```diff
@@ -51,7 +51,7 @@ else if (keuze.equals("v")){
                 System.out.println("Type een cijfer in van 0 t/m 9");
                 keuzenummer = userinput.nextInt();
                 if (keuzenummer <10 ){
```

Comment:
Let op dat je de spaties op de juiste plekken zit. Dit is conventie en leest wat makkelijker weg.
`if (keuzenummer < 10) {`